### ISSUE TITLE
Sleep Current

### ISSUE THREAD

**Issue by sslupsky** [BNNorman/ilabs-challenger-rp2040-lora#1]
I came across your review on the ILABS website.  Did you observe and record a sleep current measurement?

**Comment by BNNorman** [comment]
No, sorry. I have seen folks quote 0.39-1.0mA for RP2040 in sleep so not useful for battery operation unless you fit a high capacity battery and recharge it during daylight hours.
